### DISCUSSION TITLE
Can't delete alarm anymore

### DISCUSSION THREAD

**Discussion by Coax88** [leikoilja/ha-google-home#789] [Q&A] [answered]
am I the only one experince this, i can't delete and alarm on google speaker anymore, worked fine last week. 

![image](https://github.com/leikoilja/ha-google-home/assets/51509073/a4089d0d-0482-4c27-9d2f-33c7e385f9bc)

**Comment by KapJI** [comment, marked as answer]
Fixed by https://github.com/leikoilja/ha-google-home/pull/792
